### PR TITLE
Allow use of default TeX fonts in PGF

### PR DESCRIPTION
**Pull request by darwindarak**
This adds a `texfonts` keyword argument to `PGF` that is `false` by default.  It suppresses calls to `fontspec` (see #87).

### CODE REVIEW

**Comment by bastinat0r** [comment]
I have not tested it yet, but from reading through the diff I would say thats exactly what we LaTeX users need. Thanks :)

**Comment by dcjones** [comment]
Looks like a good idea, thanks!

File: src/pgf_backend.jl [hunk 1]
```diff
@@ -74,11 +74,14 @@ type PGF <: Backend
...
 
+    # Use default TeX fonts instead of fonts specified by the theme.
+    texfonts::Bool
     function PGF(out::IO,
                  width,
                  height,
                  emit_on_finish::Bool=true,
-                 only_tikz = false)
+                 only_tikz = false;
+                 texfonts = false;)
```
Comment (Keno) [inline src/pgf_backend.jl]: Two semicolons.

Comment (dcjones) [inline src/pgf_backend.jl]: Thanks, fixed.

**Pull request merged**